docs/clawback/clawback-of-issued-currency.md

```diff
@@ -6,18 +6,18 @@ labels:
   - Tokens
 status: not_enabled
 ---
+
```

Comment:
We intentionally _don't_ leave a blank line between the frontmatter and the h1 at the start of a page.